Makefile

```diff
@@ -42,9 +43,24 @@ nupkg: lib lib-win
 	#dotnet nuget push *.nupkg --api-key <API_KEY> --source 	https://api.nuget.org/v3/index.json
 
 .PHONY: deps
+.ONESHELL:
 deps:
+	# install gcc
 	sudo apt install gcc
 	sudo apt install gcc-mingw-w64
+	# install pcre
+	sudo apt install libpcre3-dev
+	# install swig
+	sudo apt install swig
```

Comment:
Does this install swig v4.0.2? For me it installs version `3.something`.

Reply:
For me it installs `4.0.1`. As I said, that line may be simple, but not the latest version. Do you want me to stick to the longer one, given that it is the latest version?

Reply:
It installs `v3.0.12-1` for me so I'd say let's use the longer one. Good job on finding this install method, btw!

Reply:
Done. the longer (official) instalation is provided now.